sceptre/config/reader.py

```diff
@@ -149,6 +149,15 @@ def class_constructor(loader, node):
                 )
 
     def construct_stacks(self):
+        """
+        Recurses through the command path. For each Stack that is encountered
```

Comment:
I'd suggest rephrasing the first bit to something like:

> Traverses the files under the command path. For each file encountered, a Stack is constructed using the correct config. Dependencies are traversed [...]

This is because:

- There isn't strictly any recursion (`construct_stacks` never calls itself, directly or indirectly).
- The set of paths is an implementation detail for implementing the traversal.